src/exchanges/gemini-client.js

```diff
@@ -259,6 +328,24 @@ class GeminiClient extends EventEmitter {
         }
         return;
       }
+      if (subscription.tickers) {
+        const marketId = subscription.market.id;
+        this.tickersCache[marketId] = this.tickersCache[marketId] || new TickerCache({ market: subscription.market });
+        const newAsk = msg.events.find(thisEvt => thisEvt.type === 'change' && thisEvt.side === 'ask');
+        const newBid = msg.events.find(thisEvt => thisEvt.type === 'change' && thisEvt.side === 'bid');
+        const newTrade = msg.events.find(thisEvt => thisEvt.type === 'trade');
+        if (newAsk) {
```

Comment:
grab new ask/bid/trade data from the event and update the cached ticker (since we may only get one data point in an update, no guarantee it will have all of ask, bid and trade)